services/api/.env

```diff
@@ -43,6 +43,4 @@ UPLOADS_GCS_BUCKET=bedrock-staging-uploads
 # Sentry Error Tracking
 SENTRY_DSN=
 
-LOG_LEVEL=debug
```

Comment:
perhaps we should mention something about this in the readme, note for later

Reply:
it is in the readme: https://github.com/bedrockio/bedrock-core/tree/master/services/api#logging